mobile/src/main/java/mycroft/ai/MainActivity.java

```diff
@@ -39,42 +49,51 @@
 import io.fabric.sdk.android.Fabric;
 import mycroft.ai.adapters.MycroftAdapter;
 import mycroft.ai.receivers.NetworkChangeReceiver;
+import mycroft.ai.services.MycroftWearListenerService;
```

Comment:
`MycroftWearListenerService` is only being imported to use its constants. I recommend we move those constants to this file, so this default package doesn't depend on the `services` package.

Also, add `"MYCROFT_WEAR_REQUEST"` as a constant in this file and reference it from `MycroftWearListenerService`.